src/osd/OSD.cc

```diff
@@ -11534,6 +11494,187 @@ void OSD::ShardedOpWQ::stop_for_fast_shutdown()
   }
 }
 
+// =============================================================
+
+#undef dout_context
+#define dout_context cct
+#undef dout_prefix
+#define dout_prefix *_dout << "OSDBenchTest: "
+
+int OSDBenchTest::precheck()
+{
+  int ret = 0;
+
+  if (!store) {
+    derr << "OSDBenchTest: objectstore not specified!" << dendl;
+    ret = -ENOENT;
+  }
+
+  if (!ch) {
+    derr << "OSDBenchTest: meta collection not specified!" << dendl;
+    ret = -ENOENT;
+  }
+
+  if (!count || !bsize) {
+    ret = -EINVAL;
+  }
+
+  return ret;
+}
+
+int OSDBenchTest::run_test()
+{
+  int ret = 0;
```

Comment:
combine with the next line?

Reply:
Done now. Looks like I missed actually changing it the first time around.